src/SB/Game/zNPCTypeKingJelly.cpp

```diff
@@ -1314,6 +1319,38 @@ void zNPCKingJelly::BUpdate(xVec3* pos)
     zNPCCommon::BUpdate(pos);
 }
 
+S32 zNPCKingJelly::SysEvent(xBase* from, xBase* to, U32 toEvent, const F32* toParam,
+                            xBase* toParamWidget, S32* handled)
+{
+    U32 uVar1;
+
+    if (toEvent == 0x1b9)
+    {
+    LAB_8014808c:
+        uVar1 = 1;
+    }
+    else
+    {
+        if (toEvent < 0x1b9)
+        {
+            if (toEvent == 0x1b5)
+            {
+                start_fight();
+                goto LAB_8014808c;
```

Comment:
I'm pretty sure these goto's are just `return 1;`. If this function doesn't match yet or this suggestion otherwise doesn't work for some reason we can leave it as is.